src/test/resources/org/broadinstitute/hellbender/tools/expected.clipReadsTestCRAM.QT_10.tmp

```diff
@@ -0,0 +1,11 @@
+--------------------------------------------------------------------------------
```

Comment:
Move `ClipReads`-related test files to a `clipreads` subdirectory within `src/test/resources/org/broadinstitute/hellbender/tools/`